lib/takeaway.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'menu'
+require 'twilio-ruby'
+
+class Takeaway
+
+  ACCOUNTS_SID = 'AC0d5ab4e4089ccba3ffa0d6e307f3220d'
+  AUTH_TOKEN = 'e513890d85d278d4cf1513a63d89dcaa'
+
+  attr_reader :menu, :total
+
+  def initialize(menu_klass = Menu.new, twilio_klass = Twilio::REST::Client.new(ACCOUNTS_SID, AUTH_TOKEN))
```

Comment:
Metrics/LineLength: Line is too long. [106/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)